### PR TITLE
Remove Travis; add default Ruby Github Actions CI

### PR DESCRIPTION
**Pull request by jcoleman**
travis-ci.org is long deprecated, and Github actions avoids the need for separate accounts with a different service entirely.

Also set a default Ruby version for the project and use modern Ruby versions for CI.

This replaces #4.

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+permissions:
+  contents: read
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.6', '2.7', '3.0']
```
Comment (take-five) [inline .github/workflows/ruby.yml:25]: I think Ruby 2.6 is EOL and 2.7 is almost EOL?
What do you think of `['2.7', '3.0', '3.1', '3.2']`?

**Review by take-five — APPROVED** [review]
Thank you very much for your contribution!
I left one comment. I think it's not critical, but would be nice to have.
What do you think?

File: .github/workflows/ruby.yml (hunk 1 above)
Comment (jcoleman) [reply .github/workflows/ruby.yml:25]: Yeah, that's a reasonable change. This just came from the default Workflow setup from Github, so I'm not glued to it.

Comment (jcoleman) [reply .github/workflows/ruby.yml]: Updated!

**Comment by take-five** [comment]
Thank you!

**Pull request merged**